src/main/java/com/ariatemplates/attester/maven/ArtifactExtractor.java

```diff
@@ -88,12 +96,18 @@ public static String getPropertyName(Dependency dependency) {
     }
 
     public static File getOutputDirectory(Artifact artifact) {
```

Comment:
Thank you for your remark. Indeed, this method is not used anymore in attester-maven-plugin.
However, it can be considered as part of the public API of the class, so I think we should remove it in a future (non-backward compatible) release.